tests/Paket.Tests/Resolver/GlobalOptimisticStrategySpecs.fs

```diff
@@ -159,7 +159,7 @@ let ``should favor strategy from parent that overrides strategy``() =
     let resolved =
         DependenciesFile.FromSource(config)
         |> resolve graph2 UpdateMode.UpdateAll
-    getVersion resolved.[PackageName "Castle.Windsor"] |> shouldEqual "3.2.1"
+    getVersion resolved.[PackageName "Castle.Windsor"] |> shouldEqual "3.3.0"
```

Comment:
why is this increased? I thought we use min strategy?

Reply:
this test:
```
[<Test>]
let ``should favor strategy from parent that overrides strategy``() = 
    let config = """
    strategy max
    source http://www.nuget.org/api/v2
    nuget Nancy.Bootstrappers.Windsor !~> 0.23
    nuget Castle.Windsor
    nuget Castle.Windsor-NLog !> 0
    """
    let resolved =
        DependenciesFile.FromSource(config)
        |> resolve graph2 UpdateMode.UpdateAll
    getVersion resolved.[PackageName "Castle.Windsor"] |> shouldEqual "3.3.0"
    getVersion resolved.[PackageName "Castle.Windsor-NLog"] |> shouldEqual "3.3.0"
    getVersion resolved.[PackageName "Castle.Core-NLog"] |> shouldEqual "3.3.0"
    getVersion resolved.[PackageName "Castle.Core"] |> shouldEqual "3.3.0"
    getVersion resolved.[PackageName "Nancy.Bootstrappers.Windsor"] |> shouldEqual "0.23"
```
is set to max but more importantly has Castle.Windsor as a top level dependency.  So should take the latest version